TECH.md

```diff
@@ -0,0 +1,4 @@
+OpenAPI Space is a Python API built with Flask and Connexion
```

Comment:
It's not really a "Python API", but rather a Python app that provides access to a API spec database through a REST API.